includes/admin/class-sensei-extensions.php

```diff
@@ -54,23 +54,49 @@ public function enqueue_admin_assets() {
 		if ( in_array( $screen->id, [ 'sensei-lms_page_sensei-extensions' ], true ) ) {
 			Sensei()->assets->enqueue( 'sensei-extensions', 'extensions/index.js', [], true );
 			Sensei()->assets->enqueue( 'sensei-extensions-style', 'extensions/extensions.css' );
+			$this->localize_script();
+		}
+	}
 
-			wp_localize_script(
-				'sensei-extensions',
-				'sensei_extensions',
-				[
-					'connectUrl' => add_query_arg(
-						array(
-							'page'              => 'wc-addons',
-							'section'           => 'helper',
-							'wc-helper-connect' => 1,
-							'wc-helper-nonce'   => wp_create_nonce( 'connect' ),
-						),
-						admin_url( 'admin.php' )
+	/**
+	 * Localize extensions script.
+	 *
+	 * @since 3.11.0
+	 */
+	private function localize_script() {
+		$data = array(
+			'connectUrl' => add_query_arg(
+				array(
+					'page'              => 'wc-addons',
+					'section'           => 'helper',
+					'wc-helper-connect' => 1,
+					'wc-helper-nonce'   => wp_create_nonce( 'connect' ),
+				),
+				admin_url( 'admin.php' )
+			),
+		);
+
+		if ( ! Sensei_Utils::is_woocommerce_installed() ) {
+			$data['installUrl'] = self_admin_url( 'plugin-install.php?s=woocommerce&tab=search&type=term&plugin_details=woocommerce' );
+		} elseif ( ! Sensei_Utils::is_woocommerce_active() ) {
+			$plugin_file         = 'woocommerce/woocommerce.php';
+			$data['activateUrl'] = wp_nonce_url(
+				add_query_arg(
+					array(
+						'action' => 'activate',
+						'plugin' => $plugin_file,
 					),
-				]
+					self_admin_url( 'plugins.php' )
+				),
+				'activate-plugin_' . $plugin_file
 			);
 		}
+
+		wp_localize_script(
```

Comment:
Also about the new variables. All of them could be a part of the extensions response, like also commented here: https://github.com/Automattic/sensei/pull/4213#discussion_r626890295

There are also some other WC variables coming through the extensions request that could be grouped with these ones. :)